etc/kayobe/ansible/cardiff-run.yml

```diff
@@ -0,0 +1,46 @@
+---
+- name: Cardiff run
+  hosts: localhost
+  gather_facts: no
+  tags:
+    - cardiff
+  vars:
+    venv: "~/venvs/cardiff-venv"
+    input_dir: "{{ lookup('env', 'PWD') }}/overcloud-introspection-data"
+    output_dir: "{{ lookup('env', 'PWD') }}/review"
+    cardiff_pattern: ".*.eval" # Uses regex
+  tasks:
+    - name: Install dependencies
+      pip:
+        virtualenv: "{{ venv }}"
+        name:
+          - git+https://github.com/stackhpc/cardiff
+          #- /home/matt/Documents/cardiff
+          - pyvis
+          - pyvis.network
+        state: latest
+
+    - name: Create data directory
+      file:
+        path: '{{ output_dir }}/data'
+        state: directory
+
+    - name: Extract data
+      shell:
```

Comment:
nit: could use `command`

Reply:
These two commands need to use shell for the filename expansions

Reply:
ack